src/main/resources/templates/vet/list.html

```diff
@@ -56,14 +56,14 @@
                         <img style="width:320px;height:300px;" th:attr="src=@{${gcpImageUrl} + ${defaultImage}}"/>
                     </div>
                     <div class="caption">
-                        <h3 th:text="${pet.name}"></h3>
+                        <h3 th:data-testid="petName" th:text="${pet.name}"></h3>
                         <p>
                         <ul>
-                            <li th:text="${@dateFormatter.formatToDate(pet.birthDate, #locale)}"/>
-                            <li th:text="${pet.breed.name}"/>
-                            <li th:text="${pet.dewormed}? #{pet.dewormed} : #{pet.not.dewormed}"/>
-                            <li th:text="${pet.sterilized}? #{pet.sterilized} : #{pet.not.sterilized}"/>
-                            <li th:text="${pet.vaccinated}? #{pet.vaccinated} : #{pet.not.vaccinated}"/>
+                            <li th:data-testid="petBirthdate" th:text="${@dateFormatter.formatToDate(pet.birthDate, #locale)}"/>
+                            <li th:data-testid="petBreed" th:text="${pet.breed.name}"/>
+                            <li th:data-testid="petDewormed" th:text="${pet.dewormed}? #{pet.dewormed} : #{pet.not.dewormed}"/>
```

Comment:
Let's change it for `th:data-testid="petDeworming"` to make it consistent with the another PR.